Pinta.Core/Extensions/Cairo/CairoExtensions.ColorType.cs

```diff
@@ -102,6 +102,35 @@ static string ExpandColorHex (string hex)
 		}
 	}
 
+	/// <summary>
+	/// Parses the color from a hex string in the byte order of the ColorBgra struct,
+	/// for backwards compatibility with existing settings.
+	/// </summary>
+	public static Color? ParseBgraHexString (string hex)
+	{
+		Color? result = FromHex (hex);
+
+		if (result is null)
+			return null;
+
+		// Inverse of the reordering in ToBgraHexString().
+		return new (
+			result.Value.G,
+			result.Value.B,
+			result.Value.A,
+			result.Value.R);
+	}
+
+	/// <summary>
+	/// Converts the color to a hex string in the byte order of the ColorBgra struct,
+	/// for backwards compatibility with existing settings.
+	/// </summary>
+	public static string ToBgraHexString (Color color)
```

Comment:
I'm okay with moving these out into public methods, but would have a couple suggestions:
- There should be a stronger warning about avoiding these for new code (since the ordering is very unexpected). e.g. something like `This should only be used for backwards compatibility with existing settings. New code should use ToHex()`
- Now that they are public, could you also add a simple unit test for these? Just testing a color that has different values for each component, to make sure the byte ordering translates correctly, would be fine